HW06-Annotation/src/main/java/com/akproductions/testframework/TestFramework.java

```diff
@@ -0,0 +1,83 @@
+package com.akproductions.testframework;
+
+import com.akproductions.testframework.annotations.After;
+import com.akproductions.testframework.annotations.Before;
+import com.akproductions.testframework.annotations.Test;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class TestFramework {
+    private Class clazz;
+    private HashMap<String,Integer> testResults;
+    public TestFramework (String classToTest) throws Exception {
+        clazz = Class.forName(classToTest);
+        testResults= new HashMap<String,Integer> ();
+    }
+
+    public HashMap<String,Integer> doTest () throws Exception
```

Comment:
Давайте еще улучшим и заменим HashMap на класс.
Не бойтесь добавлять новые классы.
У нас ООП. 

Например, посмотрите на такую сигнатуру метода:
```java
public TestResult doTest()
```
Сразу видно по сигнатуре, что возвращается результат тестирования.

А в случае:
```java
public HashMap<String,Integer> doTest()
```
Какая-то мапа со строкой в ключе и числом в значении.
Да, можно посмотреть, изучить код и понять, что там возвращается, но это трата лишних сил, времени и тп 
И с мапой со строковыми ключами можно сделать ошибки в них, с классом у вас будет проверка во время компиляции.

Еще момент, в таких случаях лучше определять через интерфейс:
```suggestion
    public Map<String,Integer> doTest()
```